Add Workspace mapping documentation (#1649)
astronomer/docs: astro/modify-cluster.md

- Update an existing worker node pool, including its node instance type or maximum node count.
- Create a VPC connection or a transit gateway connection between a cluster and a target VPC. See [Connect Astro to external data sources](https://docs.astronomer.io/astro/category/connect-astro).

> To modify a cluster in these ways, you'll need the following:

Reviewer: ```suggestion
When you need a cluster modification that involves Astronomer support, you'll need to provide the following information:
```

For modifications that do require downtime, such as changing your cluster's node instance type, Astronomer support will inform you of the expected impact and ask you to confirm if you want to proceed.

> To confirm a modification was completed, open the **Clusters** tab in the Cloud UI. You should see the updated configuration in the table entry for your cluster.

Reviewer: ```suggestion
To confirm a modification was completed, click **Clusters** in the Cloud UI. The updated configuration appears in the table entry for your cluster.
```
I'm not clear what is meant by "...the table entry for your cluster". Is there a specific column (**Updated**) that you can reference? You can select a specific cluster to view its information. Is this what you're referencing? If so, you should add "..and then select the cluster" to the procedure step.
